docs/authentication/enterprise-connections/oidc/custom-provider.mdx

```diff
@@ -29,33 +29,48 @@ To make the setup process easier, it's recommended to keep two browser tabs open
   1. Select **Add connection**. You will be redirected to the connection's configuration page.
 
-  The next steps involve configuring both your IdP and Clerk. Keep both dashboards open as you'll need to reference them.
-
   ### Configure your IdP
 
   1. If necessary, create a new app in your IdP.
-  1. In the connection's configuration page in the Clerk Dashboard, copy the **Authorized redirect URI**.
+  1. In the Clerk Dashboard, navigate to the connection's settings page.
+  1. Copy the **Authorized redirect URI**.
   1. Add the value to your IdP's whitelisted URLs.
 
   ### Set the Discovery Endpoint, Client ID, and Client Secret in Clerk
 
-  1. In your IdP settings, copy your application's **Discovery Endpoint**, **Client ID**, and **Client Secret**.
-  1. In the connection's configuration page in the Clerk Dashboard, paste these values in their respective fields.
-  1. Add the minimum required scopes based on the IdP's documentation. Common OIDC scopes include `openid`, `email`, and `profile`.
+  Most IdPs provide a **Discovery Endpoint** to retrieve metadata about an OIDC provider. If your IdP doesn't offer this endpoint or if you need greater control over the setup process, select **Use Manual Configuration** in the **Identity Provider Configuration** section to manually configure the connection.
+
+  1. In the Clerk Dashboard, navigate to the connection's settings page.
+  1. Copy your application's **Discovery Endpoint**, **Client ID**, and **Client Secret**.
+  1. In the connection's settings page in the Clerk Dashboard, under **Identity Provider Configuration**, paste the values in their respective fields.
+  1. Under **Scopes**, add the minimum required scopes based on the IdP's documentation if needed. Common OIDC scopes include `openid`, `email`, and `profile`.
   1. Select **Save**.
 
   ### Configure attribute mapping (optional)
 
-  If your provider returns claims in a non-standard format, use the **Attribute Mapping** section on the connection's configuration page to adjust the mapping of Clerk's user properties to match the IdP's claim attributes. Important Clerk user properties include `email`, `firstName`, and `lastName`.
+  Attribute mapping allows you to map the IdP's claims with Clerk's user properties such as the `email_verified`. OIDC Enterprise connections require the [`email_verified` claim](https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims:~:text=Section%C2%A05.7.-,email_verified,-boolean) to verify email ownership. However, some IdPs, such as Microsoft Azure Active Directory, might not return this claim or use a non-standard format.
+
+  To enable attribute mapping:
+
+  1. In the Clerk Dashboard, navigate to the **Connection** tab of the connection's settings page.
+  1. In the **Attribute Mapping** section, under the **Email address verified** field:
+
+  - If the IdPs that provide the value, enter `email_verified`.
+  - For IdPs that do not provide the value, enter `xms_edov`.
```

Comment:
let's use the same syntax as the sentence before, and use a contraction!

```suggestion
  - If the IdP doesn't provide the value, enter `xms_edov`.
```

Reply:
If it's ok, I removed this part as it is to specific to Microsoft!